toolkit/scripts/get_config_deps.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# $1 - config_file
+
+if [[ $# -ne 1 ]]
+then
+    echo "ERROR: Must provide config file to get_config_deps.sh" >&2
+    exit 1
+fi
+
+config_file="$1"
+
+if [[ ! -f "$config_file" ]]
+then
+    echo "ERROR: Config file '$config_file' does not exist" >&2
+    exit 1
+fi
+
+config_base_dir=$(dirname "$config_file")
+
+pkg_lists=$(jq -r '.SystemConfigs[]?.PackageLists[]?' $config_file)
+postinstall_scripts=$(jq -r '.SystemConfigs[]?.PostInstallScripts[]?.Path' $config_file)
+preinstall_scripts=$(jq -r '.SystemConfigs[]?.PreInstallScripts[]?.Path' $config_file)
+finalizeimg_scripts=$(jq -r '.SystemConfigs[]?.FinalizeImageScripts[]?.Path' $config_file)
+additional_files=$(jq -r '.SystemConfigs[]?.AdditionalFiles?|keys?|join("\n")' $config_file)
+config_other_files="$pkg_lists $postinstall_scripts $preinstall_scripts $finalizeimg_scripts $additional_files"
+for filename in $config_other_files
+do
+    # fix path if it's relative to config_file
+	if [ "${filename:0:1}" == "/" ]
+	then
+		echo $(realpath "$filename")
+	else
+		echo $(realpath "$config_base_dir/$filename")
+	fi
```

Comment:
Nit-pick: since we know that we're dealing with an absolute path in the first case, we can drop `realpath`.
Idea 1:
```suggestion
	if [ "${filename:0:1}" == "/" ]
	then
		echo "$filename"
	else
		echo $(realpath "$config_base_dir/$filename")
	fi
```
Idea 2:
```suggestion
	if [ "${filename:0:1}" != "/" ]
	then
		filename="$(realpath "$config_base_dir/$filename")"
	fi
	
	echo "$filename"
```